src/main/java/org/jenkinsci/plugins/casc/ConfigurationAsCode.java

```diff
@@ -80,29 +80,28 @@ private void listElements(List<Object> elements, Set<Attribute> attributes) {
         }
     }
 
-    public static List<InputStream> getConfigurationInput(String configPath) throws IOException {
-        List<InputStream> is = new LinkedList<>();
+    public static LinkedList<InputStream> getConfigurationInput(String configPath) throws IOException {
```

Comment:
better not to expose the internal data type. For sure we use internally a LinkedList, but no need to expose this implementation detail to callers. Also this method could be made private, couldn't it ?

Reply:
@ndeloof Yeah you're right, we can make it return List<InputStrem>. It's public because i'm attempting to test the static method in a unit test to ensure that `plugins.ym` is indeed the first item on the returned list given a folder of several files...there's probably a better way to do it so i'm open to suggestion on how to test that.